static/app/utils/fields/index.ts

```diff
@@ -1761,7 +1822,6 @@ export const DISCOVER_FIELDS = [
   FieldKey.SDK_VERSION,
   FieldKey.HTTP_METHOD,
   FieldKey.HTTP_REFERER,
-  FieldKey.HTTP_STATUS_CODE,
```

Comment:
Why the removal here?

Reply:
Thank you for catching! Did not do this on purpose